packages/core/application/core-application-common/src/default_app_categories.ts

```diff
@@ -7,9 +7,9 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { AppCategory } from '../types';
+import type { AppCategory } from './app_category';
 
-/** @internal */
+/** @public */
```

Comment:
This is effectively publicly used by (and exposed to) plugins, given they need to reference it to define the `category` of an app, so I fixed the visibility.